### PR TITLE
Lcars

### PR DESCRIPTION
**Pull request by ingalls**
A little bit of code cleanup

### CODE REVIEW

File: bench/dictcache.js [hunk 1]
```diff
@@ -1,37 +0,0 @@
-var Benchmark = require('benchmark');
```
Comment (ingalls) [inline bench/dictcache.js]: It's a Dawg eat dawg world out there now

File: bench/reverseItp.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 var Benchmark = require('benchmark'),
     suite = new Benchmark.Suite();
-var address = require('../lib/pure/applyaddress');
+var address = require('../lib/pure/addressitp');
```
Comment (ingalls) [inline bench/reverseItp.js]: Now that we have two address types - clusters and interpolation lines, I've renamed the itp code to better reflect what it is doing.

File: lib/indexer/indexdocs-worker.js [hunk 3]
```diff
@@ -52,14 +52,6 @@ function runChecks(doc, zoom) {
     } else if (doc.properties["carmen:geocoder_stack"] &&
         typeof doc.properties["carmen:geocoder_stack"] !== 'string') {
         return 'geocoder_stack must be a string value';
-    } else if (!doc._zxy || doc._zxy.length === 0) {
```
Comment (ingalls) [inline lib/indexer/indexdocs-worker.js]: Zoom level is already checked by the master, each slave doesn't need to check this. Also `_zxy` is on the verge of deprecation

File: lib/spatialmatch.js [hunk 4]
```diff
@@ -62,8 +62,7 @@ function spatialmatch(query, phrasematches, options, callback) {
             var maxZoom = 0;
             while (l--) maxZoom = Math.max(maxZoom, stack[l].zoom);
             coalesceOpts.centerzxy = proximity.center2zxy(
-                options.proximity[0],
```
Comment (ingalls) [inline lib/spatialmatch.js]: standardize use of lon/lat array instead of separate args.

File: lib/util/relev.js [hunk 5]
```diff
@@ -1,44 +0,0 @@
-var POW2_48 = Math.pow(2,48);
```
Comment (ingalls) [inline lib/util/relev.js]: Grepping reveals that carmen no longer uses the `relev.js` module

File: test/indexdocs-worker.test.js [hunk 6]
```diff
@@ -79,30 +79,6 @@ tape('worker.runChecks', function(assert) {
         id:1,
         properties: {
             'carmen:text':'Main Street',
-            'carmen:center': [0,0]
```
Comment (ingalls) [inline test/indexdocs-worker.test.js]: As mentioned above the worker no longer needs to test zoom level as the master already does so.

**Comment by sesam** [comment]
😴

**Comment by sesam** [comment]
(Sorry, that emote was sent by a gnome in my pocket. Wont happen again. If
I only remember to always lock he phone.)

**Comment by ingalls** [comment]
No worries! Feel free to reach out if you ever have questions about the project.

**Pull request merged**